plugins/rtmp-services/data/services.json

```diff
@@ -2038,6 +2038,22 @@
                 "x264opts": "scenecut=0"
             }
         },
+        {
+            "name": "Odysee",
+            "servers": [
+                {
+                    "name": "Default",
+                    "url": "rtmp://stream.odysee.com/live"
+                }
+            ],
+            "recommended": {
+                "keyint": 1,
+                "max video bitrate": 2500,
+                "profile": "high",
+                "x264opts": "tune=zerolatency"
```

Comment:
This destroys quality for a handful of frames gained, are you sure you want this?